requirements.txt

```diff
@@ -1,2 +1,4 @@
 requests==2.18.4
-google-cloud-storage==1.8.0
\ No newline at end of file
+google-cloud-storage==1.8.0
+tenacity==4.10.0
+git+git://github.com/broadinstitute/cromwell-tools.git@2c9518ec869cea188dc5600dee148c4d9582eca1
```

Comment:
Could we update the hash code here with git tags after the other PR is merged in the Cromwell-tools?

Reply:
Yes! This is just for testing until the cromwell-tools PR is merged in